sdcm/nemesis.py

```diff
@@ -582,19 +583,28 @@ def disrupt_nodetool_seed_decommission(self, add_node=True):
             new_seed_node.is_seed = True
             self.cluster.update_seed_provider()
 
+    @latency_calculator_decorator
```

Comment:
@fgelcer how does this decorator affects regular longevities?

Reply:
@roydahan , this decorator will collect the latency metrics for this function, but nothing will be done with it in the end (I will check if it will be saved together with other data to ES)